### PR TITLE
[WIP][glyf] Implement glyph coordinates extractor

### PR DESCRIPTION
**Pull request by ebraminio**
WIP...

### CODE REVIEW

File: src/hb-ot-glyf-table.hh [hunk 1]
```diff
@@ -275,16 +317,19 @@ struct glyf
 						 composite);
     }
 
+    static const int FLAG_ON_CURVE = 0x01;
```
Comment (behdad) [inline src/hb-ot-glyf-table.hh]: Please use (anonymous) enum instead of "static const int". We're not Java :D.

Comment (ebraminio) [reply src/hb-ot-glyf-table.hh]: :D cc: @garretrieger

**Comment by ebraminio** [comment]
Going lights off for this and decrease opened PRs number.

**Comment by behdad** [comment]
It's still a valid issue and we want to fix it. Please don't close valid issue and PRs just because of inactivity.

**Comment by ebraminio** [comment]
Useful links,

* https://github.com/behdad/glyphy/blob/73d7fbd96975249c4eb806be94b9889df839335b/src/glyphy-freetype.h#L44-L95
* https://developer.apple.com/documentation/coretext/1510921-ctfontcreatepathforglyph?language=objc
* https://docs.microsoft.com/en-us/windows/desktop/api/wingdi/nf-wingdi-getglyphoutlinew
* https://docs.microsoft.com/en-us/windows/desktop/api/wingdi/nf-wingdi-getoutlinetextmetricsw

**Comment by behdad** [comment]
Can we talk about the API here, in comments, first?

**Comment by ebraminio** [comment]
> Can we talk about the API here, in comments, first?

#1686

**Comment by ebraminio** [comment]
let's start a new PR

**Pull request closed without merging**